src/mad_spark_alt/core/simple_qadi_orchestrator.py

```
@@ -0,0 +1,410 @@
+No prompt classification, no adaptive prompts - just pure hypothesis-driven consulting.
+"""
+
+import asyncio
+import logging
+import re
+from dataclasses import dataclass, field
+from typing import Dict, List, Optional, Tuple, Any
+
+from .interfaces import (
+    GeneratedIdea,
+    IdeaGenerationRequest,
```

Ruff (0.12.2)

`.interfaces.IdeaGenerationRequest` imported but unused

Remove unused import

(F401)